https://github.com/rui314/mold/pull/135

From: Sam James <sam@gentoo.org>
Date: Tue, 14 Dec 2021 11:01:19 +0000
Subject: [PATCH 2/2] build: don't compress man page

Negligible saving and downstream, distributions usually recompress or
compress with their own specific options. Unconditionally compressing
man pages, while well intended, usually creates more hassle there.

Signed-off-by: Sam James <sam@gentoo.org>
--- a/Makefile
+++ b/Makefile
@@ -140,8 +140,6 @@ install: all
 
 	install -m 755 -d $D$(MANDIR)/man1
 	install -m 644 docs/mold.1 $D$(MANDIR)/man1
-	rm -f $D$(MANDIR)/man1/mold.1.gz
-	gzip -9 $D$(MANDIR)/man1/mold.1
 
 	ln -sf mold $D$(BINDIR)/ld.mold
 	ln -sf mold $D$(BINDIR)/ld64.mold
